workers/unity/Packages/com.improbable.gdk.mobile/Editor/LaunchMenu.cs

```diff
@@ -1,20 +1,31 @@
+using System.Collections.Generic;
+using System.Diagnostics;
 using System.IO;
 using System.Text;
+using System.Text.RegularExpressions;
+using System.Xml;
 using Improbable.Gdk.Core;
 using Improbable.Gdk.Tools;
 using UnityEditor;
 using UnityEngine;
+using Debug = UnityEngine.Debug;
 
 namespace Improbable.Gdk.Mobile
 {
     public static class LaunchMenu
     {
         private const string rootApkPath = "build";
         private static string AbsoluteApkPath => Path.GetFullPath(Path.Combine(Application.dataPath, Path.Combine("..", rootApkPath)));
+        private static string XCodeProjectPath = Path.GetFullPath(Path.Combine(Path.Combine(Application.dataPath, "..", "build", "worker", "MobileClient\\@iOS", "MobileClient\\@iOS")));
```

Comment:
No need for nested combine - can flatten it out

Reply:
completely oversaw that 🤦‍♀